applications/directory/src/components/CreateProfileForm.js

```diff
@@ -260,6 +272,20 @@ const CreateProfileForm = ({
 
   return (
     <Box>
+      {submissionError && (
+        <Alert
+          ref={alertElement}
+          id={'#alert'}
```

Comment:
Use `id="alert"` instead?

Reply:
I do not think we need it. I am deleting it